packages/eslint-plugin-turbo/lib/rules/no-undeclared-env-vars.ts

```diff
@@ -96,9 +96,12 @@ function create(context: Rule.RuleContext): Rule.RuleListener {
           // accessing key on process.env
           else if (
             "property" in node.parent &&
-            "name" in node.parent.property
+            "name" in node.parent.property &&
+            // && the property is not a variable.
+            // Types do not map to reality here.
```

Comment:
We can solve this and still make TS happy!

Reply:
I regret to inform you that I specifically tried to do this before switching to `any` and it does not work. The reason it doesn't work is that the argument for `isComputed` isn't `node`; it should be `node.parent.property` which is just `Identifier | PrivateIdentifier`.

There's no way to dig ourselves out of this hole without patching `@types/estree` so I ejected to `any` on this with protected access to ensure safety.